rust/stracciatella/src/schemas/yaml/tactical-map-item-replacements.schema.yaml

```diff
@@ -0,0 +1,25 @@
+$schema: http://json-schema.org/draft-04/schema#
+type: array
+title: Tactical Map Item Replacements
+items:
+  type: object
+  title: Tactical Map Item Replacement
+  description:
+    Mapping of Item ID for pre-placed items on maps. This allows mods to
+    replace items with something else, or ignore totally. This does not affect
+    items created from other means.
+
+    This is intended for compatibility and for maps that was originally built 
+    for different item sets (e.g. Wildfire maps).
+
+    Items are identified by either item index (if given integer), or internal 
+    name (if given string). Items mapped to 0 ("Nothing") are removed from map 
+    placements.
+  properties:
+    from:
+      $ref: types/id.schema.yaml
```

Comment:
Both `from` and `to` values can be numeric ID or internal name. It is because we need to replace items which does not even exist in our config or with an invalid ID. This helps supporting maps ported from 1.13.

See also: https://github.com/ja2-stracciatella/ja2-stracciatella/blob/e23678b74af7e7fa2ccaf067eb44993184af0a33/src/externalized/tactical/MapItemReplacementModel.cc#L3

Reply:
Fixed. Thanks for the catch!